apps/trustlab/src/components/Banner/Banner.js

```diff
@@ -0,0 +1,85 @@
+import { Section, RichTypography } from "@commons-ui/core";
+import { Figure } from "@commons-ui/next";
+import { LexicalRichText } from "@commons-ui/payload";
+import { Box, Grid } from "@mui/material";
+
+function Banner({
+  backgroundColor,
+  textColour,
+  title,
+  description,
+  image,
+  isPageHeader,
+}) {
+  const hasImage = image && image.url;
+
+  return (
+    <Section
+      sx={{
+        maxWidth: { md: "100%", xs: "100%" },
+        backgroundColor,
+      }}
+      fixed={false}
+    >
+      <Box
```

Comment:
Aren't we using this in reverse? Section is supposed to handle maxWidth uniformly for the whole site. If we want something that is always 100% wide, why not:

1. `<Box><Section...`
2. Just `Section` with `before` [pseudo-element](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements) to sent background at 100% width, etc.